client/extensions/woocommerce/components/product-search/variations.js

```diff
@@ -88,12 +86,9 @@ class ProductVariations extends Component {
 			<div className="product-search__variations">
 				<FormFieldset>
 					<FormLegend>
-						<Notice showDismiss={ false }>
-							{ translate(
-								'%(product)s has variations. Choose a specific customization to select.',
-								{ args: { product: product.name } }
-							) }
-						</Notice>
+						{ translate( '%(product)s has variations. Choose a specific customization to select.', {
```

Comment:
How about instead of `Choose a specific customization to select.` we have `Choose a specific variation to add.` This way we avoid introducing a new term (customization).